subsys/bluetooth/mesh/proxy.h

```diff
@@ -6,20 +6,21 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#ifndef ZEPHYR_SUBSYS_BLUETOOTH_MESH_PROXY_H_
+#define ZEPHYR_SUBSYS_BLUETOOTH_MESH_PROXY_H_
+
+#ifdef CONFIG_BT_MESH_PROXY
+#include <bluetooth/gatt.h>
+
+int bt_mesh_pb_gatt_send(struct bt_conn *conn, struct net_buf_simple *buf,
+			 bt_gatt_complete_func_t end, void *user_data);
+#endif /* CONFIG_BT_MESH_PROXY */
```

Comment:
If we place `bt_gatt_complete_func_t` to proxy.h, must include `gatt.h`, and proxy.h has been include by other where. Another constraint is that the use of gatt.h must be enable config_bt_conn.

``` C
/* gatt.h */

#define BT_GATT_CCC_MAX (CONFIG_BT_MAX_PAIRED + CONFIG_BT_MAX_CONN)
```

Therefore, I’m here to determine whether it include.

Reply:
@MarekPieta FYI, are you working on this particular change?

Reply:
I will open a PR with fix today.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/34133